conda/dev.yaml

```diff
@@ -11,4 +11,5 @@ dependencies:
   - pip
   - pip:
       # distlib issue
-      - paginate
+      - distlib
```

Comment:
we had this in the past, but it didn't work very stable .. sometimes it worked .. sometimes it didn't ... that is a very weird non-deterministic issue  .. we can keep it for now

Reply:
could you move back the changes of this file? with the new changes form main, it shouldn't be necessary

Reply:
could you move back the changes of this file? with the new changes form main, it shouldn't be necessary